lib/Doctrine/DBAL/Driver/PDOStatement.php

```diff
@@ -85,7 +85,13 @@ public function bindValue($param, $value, $type = ParameterType::STRING)
     }
 
     /**
-     * {@inheritdoc}
+     * @param mixed    $column
+     * @param mixed    $variable
+     * @param int      $type
+     * @param int|null $length
+     * @param mixed    $driverOptions
```

Comment:
Although it's [documented](https://www.php.net/manual/en/pdostatement.bindparam) as `mixed`, I've never seen it be anything else than `array`. Most likely, it's a documentation bug. Should we declare it as `array`?

Reply:
The usage examples in the same manual contain `$driver_options = PDO::SQLSRV_ENCODING_BINARY` so it looks like it's really `mixed`.